src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -29,102 +31,83 @@ function transformRelationship(obj) {
   return rel;
 }
 
+// For each applicant, adjust organization of the object, add supporting documents array
 function transformApplicants(applicants) {
   const applicantsPostTransform = [];
-
   applicants.forEach(app => {
-    const transformedApp = {
-      fullName: app.applicantName ?? '',
+    let transformedApp = {
+      ...app,
       ssnOrTin: app.applicantSSN?.ssn ?? '',
-      dateOfBirth: fmtDate(app.applicantDob) ?? '',
-      phoneNumber: app.applicantPhone ?? '',
-      email: app.applicantEmailAddress ?? '',
       vetRelationship: transformRelationship(
         app.applicantRelationshipToSponsor || 'NA',
       ),
-      // sponsorMarriageDetails:
-      // app?.applicantSponsorMarriageDetails?.relationshipToVeteran || 'NA',
-      isEnrolledInMedicare:
-        app?.applicantMedicareStatus?.eligibility === 'enrolled',
-      hasOtherHealthInsurance: app?.applicantHasOhi?.hasOhi === 'yes',
-      applicantSupportingDocuments: [
-        app?.applicantMedicareCardFront,
-        app?.applicantMedicareCardBack,
-        app?.applicantOHICardFront,
-        app?.applicantOHICardBack,
-        app?.applicantBirthCertOrSocialSecCard,
-        app?.applicantSchoolCert,
-        app?.applicantAdoptionPapers,
-        app?.applicantStepMarriageCert,
-        app?.applicantMarriageCert,
-        // app?.applicantSecondMarriageCert,
-        // app?.applicantSecondMarriageDivorceCert,
-        app?.applicantMedicarePartAPartBCard,
-        app?.applicantMedicarePartDCard,
-        app?.applicantMedicareIneligibleProof,
-        app?.applicantOhiCard,
-        app?.applicantOtherInsuranceCertification,
-        app?.applicantHelplessCert,
-      ],
-      address: app.applicantAddress ?? {},
-      gender: app.applicantGender?.gender ?? '',
+      // Grab any file upload properties from this applicant and combine into a
+      // supporting documents array:
+      applicantSupportingDocuments: Object.keys({
+        ...REQUIRED_FILES,
+        ...OPTIONAL_FILES,
+      })
+        .filter(k => k.includes('applicant')) // Ignore sponsor files
+        .map(f => app?.[f]) // Grab the upload obj from top-level in applicant
+        .filter(el => el), // Drop any undefineds/nulls
     };
-
-    // eslint-disable-next-line dot-notation
-    transformedApp.address['postal_code'] = transformedApp.address.postalCode;
-    delete transformedApp.address.postalCode;
-
+    transformedApp = adjustYearString(transformedApp);
+    transformedApp.applicantAddress.streetCombined = concatStreets(
+      transformedApp.applicantAddress,
+    );
```

Comment:
Using new helper functions to adjust date format and concatenate all 3 street names into one string